lang/java.md

```diff
@@ -0,0 +1,114 @@
+---
+    lang: Java
+    syntax: java
+    title: Java is a statically typed JVM based programming language.
+    author: Khafia Ayyub
+---
+
+```java
+    public class HelloWorld {
+        public static void main(String[] args) {
+            System.out.println("Hello, World!"); 
+        }
+    }
+```
+
+## What is Java?
+
+Java is a programming language and computing platform first released by Sun Microsystems in 1995. There are lots of applications and websites that will not work unless you have 
+Java installed, and more are created every day. Java is a general-purpose, class-based, object-oriented programming language designed for having lesser implementation dependencies. 
+It is a computing platform for application development. The compilation of the Java applications results in the bytecode that can be run on any platform using the 
+Java Virtual Machine(JVM). Java is fast, secure, and reliable. From laptops to datacenters, game consoles to scientific supercomputers, cell phones to the Internet, Java is 
+everywhere!
+
+
+## What is Java used for?
+
+**Here are some important Java applications:-**
+
+✦ It is used for developing Android Apps
```

Comment:
Please use a valid markdown syntax to display a list. This won't be rendered properly on the website. Either go with an unordered list or a numbered list.